### ISSUE TITLE
`conPty` does not release stdin/terminal

### ISSUE THREAD

**Issue by louis-bompart** [microsoft/node-pty#471]
## Environment details

- OS: Windows 10
- OS version: 10.0.19041
- node-pty version: 0.10.0
- Node: v14.16.0

## Issue description

When running `node-pty` with `useConpty:true`  -or unset, for my Windows version at least-, the node process does not finish.
This does not occurs with `useConpty:false`.

To help with the reproduction of the issue, I made an MRE-repo [here](https://github.com/louis-bompart/automatic-dollop).

**Comment by louis-bompart** [comment]
Hey @Tyriar, nice thanks!
Just to check, the issue should be fixed in https://www.npmjs.com/package/node-pty/v/0.11.0-beta9 (and then `0.11.0`)?

**Comment by Tyriar** [comment]
Yes that's right, 0.11.0 or 1.0.0, whichever comes first. In the meantime using the beta build should be fine, if you want to be cautious the build shipped to all VS Code users is beta7: https://github.com/microsoft/vscode/blob/c416a017862f5bbd4020b5a1326441a1df7d4bb5/package.json#L77
